src/npcmove.cpp

```diff
@@ -4012,7 +4012,7 @@ bool npc::do_player_activity()
         }
     }
     if( multi_type && moves == moves_before ) {
-        moves -= 1;
+        mod_moves( -1 );
```

Comment:
same question

Reply:
Again no idea what this one is doing but I am assuming it is doing something useful and there is a need to recalculate.

If we did get speed * 0.01 and their speed was <100, you'd get a decimal result which is truncated down to 0. So that would actually change the behavior. If modifying the moves *is* doing something useful, then that would be an unwise change.